go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/tektoncd/triggers
 
-go 1.21
+go 1.22
+
+toolchain go1.22.5
```

Comment:
Do we need this toolchain? I think this might break things. I have this being added.

Reply:
It should work. Let's rebase the pr.